study-builder/studies/pancan/emails/emails.en

```diff
@@ -0,0 +1,9 @@
+#set( $globalTranslationsFile = {
+    "salutation": "Dear Participant/Parent,",
```

Comment:
not sure it should be like that. I believe it should be Dear Participant when there is no governed user and Dear Parent when there is a governed user

Reply:
Fixed.